ballerina-shell/modules/shell-core/src/main/java/io/ballerina/shell/parser/trials/ModuleMemberTrial.java

```diff
@@ -60,4 +66,18 @@ public Node parseSource(String source) throws ParserTrialFailedException {
         }
         return dclnNode;
     }
+
+    private void validateModuleDeclaration(ModuleMemberDeclarationNode declarationNode) throws InvalidMethodException {
```

Comment:
`InvalidMethodException` is not a checked exception.

Reply:
Updated in fb1b7b1aa02f28366be706cd2d53dddafb176c76